src/tests/Parser/Brewmaster/SimpleFight.js

```diff
@@ -0,0 +1,88 @@
+import SPELLS from 'common/SPELLS';
+
+export const FIGHT_END = 11000;
+export const TOTAL_STAGGERED = 240;
+
+const thisPlayer = 1;
+const enemy = 2;
+
+export function processEvents(events, module, fightEndTime) {
+  if (!fightEndTime) {
+    fightEndTime = FIGHT_END;
+  }
+  events.forEach(event => {
+    if (event.timestamp <= fightEndTime) {
+      if (event.type === "absorbed") {
```

Comment:
Please use the `fireEvent` function of the module for firing the events. iirc it's also being tested so it should work reliably enough. This reduces the amount of setup code needed in tests.